app/javascript/modules/listingDetailsAside/ListingDetailsApplicationDate.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+import { RailsListing } from "../listings/SharedHelpers"
+import dayjs from "dayjs"
+import { ApplicationStatus, ApplicationStatusType, t } from "@bloom-housing/ui-components"
+import bloomTheme from "../../../../tailwind.config"
+import { isOpen } from "../../util/listingUtil"
+
+export interface ListingDetailsApplicationDateProps {
+  listing: RailsListing
+}
+
+export const ListingDetailsApplicationDate = ({ listing }: ListingDetailsApplicationDateProps) => {
+  const isApplicationOpen = isOpen(listing)
```

Comment:
oop I see we're already using this elsewhere! in that case we should move it into the parent 😁